include/triton/Dialect/TritonLang/IR/TritonLangDialect.td

```diff
@@ -0,0 +1,31 @@
+#ifndef TRITON_LANG_DIALECT
+#define TRITON_LANG_DIALECT
+
+include "mlir/IR/OpBase.td"
+
```

Comment:
Here, the TritonLang dialect reuses the types and ops from Triton dialect.
It simply adds several additional ops (and types later) for representing dynamic shape (and buffer type later).